src/ArtifactBuildRestClient.ts

```diff
@@ -1,92 +1,33 @@
-import { IVssRestClientOptions } from 'azure-devops-extension-api';
-import { Build, BuildArtifact } from 'azure-devops-extension-api/Build'
-import { RestClientBase } from 'azure-devops-extension-api/Common/RestClientBase'
+import { BuildRestClient } from 'azure-devops-extension-api/Build'
+import { getAccessToken } from 'azure-devops-extension-sdk';
 
-export type ArtifactBuildRestClient = Pick<BuildRestClient2, 'getArtifacts' | 'getArtifactContentZip'>
+export type ArtifactBuildRestClient = Pick<BuildRestClient, 'getArtifacts'>
 
-export class BuildRestClient2 extends RestClientBase {
-    constructor(options: IVssRestClientOptions) {
-        super(options);
-    }
+export async function getArtifactContentZip(downloadUrl: string): Promise<ArrayBuffer> {
+    const accessToken = await getAccessToken();    
+    const acceptType = "application/zip";
+    const acceptHeaderValue = `${acceptType};excludeUrls=true;enumsAsNumbers=true;msDateFormat=true;noArrayWrap=true`;
```

Comment:
Do you need an apiVersion here? (I assume not since you're making this PR :D)

Reply:
Nope, it has no effect